app/views/schools/choose_programme/advisory.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: root_url) # what is the back path here, could we use an internal referer helper?
+%>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Do you know how you want to run your statutory induction programme?</h1>
+      <p class="govuk-body">You will be choosing a programme for any new early career teachers. They will complete this programme over 2 years.</p>
+      <p class="govuk-body">Only use this service if your school wants to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>use an approved training provider</li>
+      </ul>
+      <p class="govuk-body">Or</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>use the DfE accredited training materials</li>
+      </ul>
```

Comment:
Is the markup here definitely right? This doesn't feel like two lists semantically, and I wonder how a screenreader would deal with this

Reply:
LGTM, open to suggestions ?

Reply:
I would go with:

```
      <ul class="govuk-list govuk-list--bullet">
        <li>use an approved training provider; or</li>
        <li>use the DfE accredited training materials</li>
      </ul>
```

However, that's a copy change which would probably require approval from whoever wrote the copy in the first place

Reply:
open to suggestions? LGTM